modules/library/api/src/test/java/org/geotools/styling/AnchorPointTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2017, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.styling;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ *
+ * @source $URL$
+ */
+public class AnchorPointTest {
+
+  @Test
+  public void testDefaultAnchorPoint() {
+    AnchorPoint anchorPoint = AnchorPoint.DEFAULT;
+
+    assertEquals(0.0, anchorPoint.getAnchorPointX().evaluate(null, Double.class), 0.0);
+    assertEquals(0.5, anchorPoint.getAnchorPointY().evaluate(null, Double.class), 0.0);
+  }
```

Comment:
Test looks good, hold I thought only one of the values was 0.0.

From https://osgeo-org.atlassian.net/browse/GEOT-5632

> According to SLD v1.0 standard on page 47,
> The default point is X=0, Y=0.5, which is at the middle height of the lefthand
side of the label.